repairnator/src/main/java/fr/inria/spirals/repairnator/serializer/json/JsonSerializer.java

```diff
@@ -326,7 +326,8 @@ public void serializeData(ProjectInspector inspector) {
                 outputHasBeenPatchedInspector(inspector);
                 break;
 
-            case FIXERBUILD:
+            case FIXERBUILD_CASE1:
+            case FIXERBUILD_CASE2:
```

Comment:
I treated all of the other cases as the same thing because I have to think about the `ProjectState`s I really need.